web/package.json

```diff
@@ -53,12 +54,16 @@
     "@types/react-dom": "18.0.5",
     "@types/react-table": "^7.7.11",
     "@vitejs/plugin-react": "^1.3.2",
+    "aws-amplify": "^4.3.27",
+    "dotenv": "^16.0.1",
     "eslint": "8.13.0",
     "eslint-config-next": "12.1.5",
+    "fs": "0.0.1-security",
```

Comment:
`fs` is built in to Node so you shouldn't need to install it - we can remove this

Reply:
Cheers Chris, amended